music21/note.py

```diff
@@ -1190,6 +1192,22 @@ def _setVolume(self, value, setClient=True):
         [120, 80]
         ''')
 
+    def _getStoredInstrument(self):
+        return self._storedInstrument
+
+    def _setStoredInstrument(self, newValue):
+        self._storedInstrument = newValue
+
+    storedInstrument = property(_getStoredInstrument, _setStoredInstrument)
```

Comment:
I must have done this a while back when trying to emulate other m21 properties. This should probably just be an attribute.

Reply:
Oh I see, I merely moved this from Unpitched without changing it.

Reply:
yeah -- I didn't know back then how easy it was to change an attribute to a property later.

Reply:
but let's leave it.  We might want to change the displayStep etc. based on the instrument.